Flank/flank
# feat: Add storing output as json

Fixes #1437 

## Test Plan
> How do we know the code works?

1. Enable output report using option ( cli: `--output-report=json`, yaml: `output-report: json` under `flank` root)
1. Run Flank and see that the output reported is stored ( file `outputReport.json`)
1. Compare file content with console output

## Checklist

- [x] [SDD](https://github.com/Flank/flank/issues/1437#issuecomment-769299621)
